content/departments/technical-success/ce/process/license_keys.md

```diff
@@ -111,6 +126,7 @@ First the tags that relate to license itself:
 - `plg-trial` to indicate that a Cloud trial managed instance has been requested through signup.sourcegraph.com.
 - `dev` for internal developer licenses
 - `internal` for licenses used for internal sites (dotcom, k8s, etc.)
+- `allow-air-gapped` for licenses where periodic license verification is not required. This is essential for customers that run Sourcegraph in a sandboxed environment without an external internet connection.
```

Comment:
```suggestion
- `allow-air-gapped` for licenses where periodic license verification is not required. This is essential for customers that run Sourcegraph in a sandboxed environment without an external internet connection. For `enterprise-air-gap-0` plan, the tag is not needed, since the plan already has the feature.
```